### ISSUE TITLE
Customisation Notification - is it possible (perhaps in a future version)?

### ISSUE THREAD

**Issue by danueb1991** [louislam/uptime-kuma#975]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

New Notification

### 🔖 Feature description

If a service is offline, you receive the following notification (e.g. via Telegram):
[>>NAME OF SERVICE <<] [🔴 Down] Request timeout

When a service is online again, you receive the following notification (e.g. via Telegram):
[>> NAME OF SERVICE <<] [✅ Up]

### ✔️ Solution

It would be nice if you could customise the content of the notification to your liking via the Uptime Kuma user interface.

**Comment by deefdragon** [comment]
I am currently working on notification templating in #646, but it is blocked by writing tests for the notification services.

**Comment by gersonporto** [comment]
This is very interesting!

**Comment by chakflying** [comment]
Duplicate of #646
